conf/etc/consul_watcher_stop_io.json

```diff
@@ -0,0 +1,9 @@
+{
+    "watches": [
+          {
+            "type": "keyprefix",
+            "prefix": "cortx/base/ha/obj/node/",
```

Comment:
I think, we have already discussed it but want to confirm that there is no other key created with this prefix in our system?

Reply:
As discussed, we are using the prefix "cortx/base/ha/obj/node/" to call the ha_stop_io script and then
 in the script itself, we have a filter for "action"==  "stop_io"
So we are good

Reply:
Yeah, other keys written by the decision maker would have same key-prefix, I think.